public/data/vasantisuthar.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "VASANTI SUTHAR",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/vasantisuthar.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vasantisuthar",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vasanti_suthar",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Showwcase",
+      "url": "https://www.showwcase.com/vasanti-suthar",
+      "icon": "showwcase"
```

Comment:
As Emma said, we don't have a Showwcase icon. Best to open an issue [**here**](https://github.com/EddieHubCommunity/LinkFree/issues/choose), we discuss, and then we decide to add it or not. Let us know if you need any help. :)

Reply:
The issue for the same is already opened so can I work on it.

Reply:
> The issue for the same is already opened so can I work on it.

#1043